yapapi/engine.py

```diff
@@ -287,6 +293,18 @@ def report_shutdown(*exc_info):
     async def add_to_async_context(self, async_context_manager: AsyncContextManager) -> None:
         await self._stack.enter_async_context(async_context_manager)
 
+    async def yagna_version_less_than(self, checked_version: str) -> bool:
```

Comment:
-> https://docs.python.org/3/library/functools.html#functools.cache

Reply:
> -> https://docs.python.org/3/library/functools.html#functools.cache

I used `functools.cache` initially, but it doesn't work with `async`.